src/shared/components/RandomBeer/RandomBeer.jsx

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import { saveItemInLocalStorage, getItemFromLocalStorage } from '../../../services/localStorage';
 import Beer from '../Beer/Beer';
 import FavouriteButton from '../FavouriteButton/FavouriteButton';
+import PropTypes from 'prop-types';
 import './styles.scss';
 
 const RandomBeer = ({ value }) => {
```

Comment:
What is value? This name doesn't tell much

Reply:
Fixed!